packages/core/components/ContextMenu/items.ts

```diff
@@ -92,6 +93,14 @@ export default function getContextMenuItems(dispatch: Dispatch) {
                     dispatch(interaction.actions.showManifestDownloadDialog());
                 },
             },
+            {
+                key: ContextMenuActions.DOWNLOAD,
+                text: "Download",
```

Comment:
Tiny quality of life improvement if it's easy: we could do something like this here: `fileSelection.length > 1 ? "Download all" : "Download"`

Reply:
For the context menu stuff we have been just following what file browsers (like MacOS Finder or even Google Drive does) and I think they don't make a distinction for downloading one or multiple items. If we did update it I feel like not also appending "all" to "Open" & "Open with" might be misleading potentially? Let me know your thoughts!

Here is Google Drive for example:
<img width="379" alt="Screen Shot 2023-03-07 at 11 33 19 AM" src="https://user-images.githubusercontent.com/41307451/223533331-ddee968d-a32e-4293-aa13-27cdf071da7f.png">

Reply:
Borrowing from the Apple/Google UX teams seems like a great strategy! Let's keep it